### DISCUSSION TITLE
加载自己的csv数据回测，输出的文件很奇怪

### DISCUSSION THREAD

**Discussion by puddingsmall** [wondertrader/wondertrader#45] [其他/General]
![image](https://user-images.githubusercontent.com/59089657/161178069-04c6435f-722b-4b6c-b394-e79c57714579.png)
![image](https://user-images.githubusercontent.com/59089657/161178129-44c01783-16a0-4f5e-89fe-3d0a18e25b7c.png)
![image](https://user-images.githubusercontent.com/59089657/161178303-cc027f76-e810-4cc3-8f53-001e1d04e107.png)

**Comment by ZzzzHeJ** [comment]
你看一下out_bt中的fund和close是否有异常

**Reply by ZzzzHeJ** [reply]
建议排查流程：
1 检查outputs_bt中输出的fund和closes是否异常
2 更换为同引擎的demo策略运行，结果是否正确
3 检查策略逻辑是否正常

**Reply by puddingsmall** [reply]
好的，我测试一下

**Comment by puddingsmall** [comment]
![image](https://user-images.githubusercontent.com/59089657/161212975-8c22252f-294a-4bb0-b96a-8c9f446152f6.png)
解决了，要删除本地生成dsb文件，然后重新导入csv数据

**Reply by ZzzzHeJ** [reply]
wt中的数据都是用his中的dsb数据的，如果没有，会根据csv中的数据自动转换，所以如果改了csv，也要把对应的dsb数据删了才会有效。
